tasks/sep/025-mt-cannon/VALIDATION.md

```diff
@@ -0,0 +1,143 @@
+  **Meaning:** This is `owners[0xca11bde05977b3631167028862be2a173976ca11] -> 1`, so the key can be
+  derived from `cast index address 0xca11bde05977b3631167028862be2a173976ca11 2`.
+
+- **Key:** 0xe90b7bceb6e7df5418fb78d8ee546e97c83a08bbccc01a0644d599ccd2a7c2e0 <br/>
+  **Value:** 0x000000000000000000000000ca11bde05977b3631167028862be2a173976ca11 <br/>
+  **Meaning:** This is `owners[1] -> 0xca11bde05977b3631167028862be2a173976ca11`, so the key can be
+  derived from `cast index address 0x0000000000000000000000000000000000000001 2`.
+
+## State Changes
+
+### `0x05F9613aDB30026FFd634f38e5C4dFd30a197Fa1` (`DisputeGameFactoryProxy`)
+
+- **Key**: `0xffdfc1249c027f9191656349feb0761381bb32c9f557e01f419fd08754bf5a1b` <br/>
+  **Before**: `0x0000000000000000000000005e0877a8f6692ed470013e651c4357d0c4941e6c` <br/>
+  **After**: `0x000000000000000000000000924D3d3B3b16E74bAb577e50d23b2a38990dD52C` <br/>
+  **Meaning**: Updates the implementation for game type 0.
+  - The [`gameImpls` mapping](https://github.com/ethereum-optimism/optimism/blob/op-contracts/v1.9.0-rc.3/packages/contracts-bedrock/src/dispute/DisputeGameFactory.sol#L57) is at [storage slot 101](https://github.com/ethereum-optimism/optimism/blob/op-contracts/v1.9.0-rc.3/packages/contracts-bedrock/snapshots/storageLayout/DisputeGameFactory.json#L41) and is keyed by [`GameType` (`uint32`)](https://github.com/ethereum-optimism/optimism/blob/op-contracts/v1.9.0-rc.3/packages/contracts-bedrock/src/dispute/lib/LibUDT.sol#L224).
+  - Confirm the expected key slot with the following:
+    ``` shell
+    cast index uint32 0 101
+    0xffdfc1249c027f9191656349feb0761381bb32c9f557e01f419fd08754bf5a1b
+    ```
+
+### `0x05F9613aDB30026FFd634f38e5C4dFd30a197Fa1` (`DisputeGameFactoryProxy`)
+
+- **Key**: `0x4d5a9bd2e41301728d41c8e705190becb4e74abe869f75bdb405b63716a35f9e` <br/>
+  **Before**: `0x0000000000000000000000004ed046e66c96600dae1a4ec39267bb0ce476e8cc` <br/>
+  **After**: `0x000000000000000000000000879e899523bA9a4Ab212a2d70cF1af73B906CbE5` <br/>
+  **Meaning**: Updates the implementation for game type 1.
+  - The [`gameImpls` mapping](https://github.com/ethereum-optimism/optimism/blob/op-contracts/v1.9.0-rc.3/packages/contracts-bedrock/src/dispute/DisputeGameFactory.sol#L57) is at [storage slot 101](https://github.com/ethereum-optimism/optimism/blob/op-contracts/v1.9.0-rc.3/packages/contracts-bedrock/snapshots/storageLayout/DisputeGameFactory.json#L41) and is keyed by [`GameType` (`uint32`)](https://github.com/ethereum-optimism/optimism/blob/op-contracts/v1.9.0-rc.3/packages/contracts-bedrock/src/dispute/lib/LibUDT.sol#L224).
+  - Confirm the expected key slot with the following:
+    ``` shell
+    cast index uint32 1 101
+    0x4d5a9bd2e41301728d41c8e705190becb4e74abe869f75bdb405b63716a35f9e
+    ```
+
+### For the Council:
+
+- **Key:** `0x2f3f3bfd9c6f0afcf70fd2d0651e7e3d6cfa084f5cfe749581936c902a946aad` <br/>
```

Comment:
Should be made clear what address/contract to look for this change in.

Same goes for the Foundation step below.